drivers/lkss/lab1/ex3/timer.c

```diff
@@ -0,0 +1,45 @@
+
+#include <linux/module.h>
+#include <linux/kernel.h>
+#include <linux/init.h>
+#include <linux/timer.h>
+#include <linux/jiffies.h>
+
+#define TIMER_INTERVAL_MS 1000  /* 1000 milliseconds = 1 second */
+
+static struct timer_list my_timer;
+
+/* Timer callback */
+static void my_timer_callback(struct timer_list *t)
+{
+	pr_info("Timer callback executed at jiffies=%lu\n", jiffies);
+	/* TODO schedule the timer again ... */
+
```

Comment:
delete?

Reply:
hmm, we want to reschedule the timer here. `deletion` of the timer is done at module exit.